src/sentry/api/endpoints/organization_events_distribution.py

```diff
@@ -56,14 +56,14 @@ def get(self, request, organization):
             referrer="api.organization-events-distribution",
         )["data"]
 
-        projects = {p.id: p.slug for p in self.get_projects(request, organization)}
-
         if key == PROJECT_KEY:
+            projects = {p.id: p.slug for p in self.get_projects(request, organization)}
+
             resp = {
                 "key": PROJECT_KEY,
                 "topValues": [
                     {
-                        "value": projects[v["project_id"]],
+                        "value": v["project_id"],
```

Comment:
I don't think we want to expose numeric ids to users if possible.

Reply:
@markstory this is kinda necessary, since we use this to add/update the `project` query string.

Reply:
Ok. That makes sense then.